### PR TITLE
new storage structure

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

this PR is to add a correct structure to the riot rules.

### CODE REVIEW

File: firmware/sys/storage/Makefile.dep [hunk 1]
```diff
@@ -1 +1,6 @@
-USEMODULE += mtd_flashpage
+  USEMODULE += mtd_flashpage
```
Comment (eduazocar) [inline firmware/sys/storage/Makefile.dep:1]:
```suggestion
USEMODULE += mtd_flashpage
```

Comment (xkevin190) [reply firmware/sys/storage/Makefile.dep:1]: what?

File: firmware/sys/storage/include/storage.h [hunk 2]
```diff
@@ -28,11 +28,18 @@
 #define STORAGE_H
 
 #include <stdio.h>
+#include "mtd_flashpage.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define last_page 1
```
Comment (eduazocar) [inline firmware/sys/storage/include/storage.h:37]: All macros should be written in uppercaps

File: firmware/sys/storage/include/storage.h [hunk 3]
```diff
@@ -28,11 +28,18 @@
...
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define    LAST_PAGE 1
+
+/**
+ * @brief this is the number of pages that we can write
+ */
+#define LAST_AVAILABLE_PAGE (FLASHPAGE_NUMOF - last_page)
```
Comment (eduazocar) [inline firmware/sys/storage/include/storage.h:42]: Ok looks better the macro in uppercaps, but the same change, should be here with last_page

File: firmware/sys/storage/Makefile.dep (hunk 1 above)
Comment (eduazocar) [reply firmware/sys/storage/Makefile.dep]: only format change jejeje

**Review by eduazocar — APPROVED** [review]
:+1: go ahead

**Pull request merged**